questions.md

```diff
@@ -0,0 +1,3 @@
+* What is a pull request?
+* What happens if a pull request has a conflict with another file someone else modified?
```

Comment:
If a Pull Request has a conflict, GitHub will notify you within the Pull Request. You would have to fix the conflict before you can merge the code changes.